### DISCUSSION TITLE
gpt-4-1106-preview is always the selected openai model after login

### DISCUSSION THREAD

**Discussion by edsonmedina** [danny-avila/LibreChat#4021] [Troubleshooting]
### What happened?

Every time I login into Librechat gpt-4-1106-preview gets selected as the default model. This happens every time the session expires and I have no login back again.

This model is outdated and much more expensive than the gpt-4o line.

gpt-4-1106-preview - input: $10.00 / 1M tokens, output: $30.00 / 1M tokens
gpt-4o - input: $5.00 / 1M input tokens, output: $15.00 / 1M output tokens (half the cost in both directions)
gpt-4o-2024-08-06 - input: $2.50 / 1M input tokens, output: $10.00 / 1M output tokens (1/4 the input cost, 1/3 the output cost)

### Steps to Reproduce

1. Select any openAI model (ie: gpt-4o)
2. Click on the user name (bottom left of the screen) and click "log out"
3. Enter your login again
4. The selected model is now gpt-4-1106-preview

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is the default order OpenAI returns the list of models in. I would like to add an option to configure the order more easily, but for now, you can specify exactly which models you want listed, in the exact order you want them.

```.env
# in your .env file
OPENAI_MODELS=gpt-4o,gpt-4o-mini # ...etc
```

**Comment by edsonmedina** [comment]
That solved it, thanks
